<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
        <param name="state_count_threshold" value="3" type="int"/>
        <param name="model_name" value="sim.pb" />
        <param name="light_detector" value="true" type="bool"/>
        <param name="tf_optimize" value="true" type="bool"/>
        <param name="debug_window" value="true" type="bool"/>

    </node>

    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
            <remap from="image" to="/dbg/traffic_light_state" />
    </node>
</launch>
